project/giter8.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.14.0")
+addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.16.0")
```

Comment:
I believe this sbt-giter8 should be, as of two months ago, 0.16.1.  If there was a reason for the G8 folks to go to .1,
probably worth using it.

Reply:
Thanks.